doc/sphinx-guides/source/developers/containers.rst

```diff
@@ -194,6 +194,115 @@ Once you have multiple Glassfish servers you may notice bugs that will require a
+```
+mkdir /tmp/share           # Create a directory that could be mounted in container. This is mounted as /share in container.
+chcon -R -t svirt_sandbox_file_t /tmp/share/ # Change selinux label so that containers can read/write from/to directory.
+chmod 777 -R /tmp/share
+oc login -u system:admin
+oc edit scc restricted     # Update allowHostDirVolumePlugin to true and runAsUser type to RunAsAny
+```
+
+To add a persistent volume and persistent volume claim, in conf/docker/openshift/openshift.json, add the following to objects in openshift.json:
+```
+    {
+      "kind" : "PersistentVolume",
+      "apiVersion" : "v1",
+      "metadata":{
+        "name" : "solr-index-backup",
+        "labels":{
+          "name" : "solr-index-backup",
+          "type" : "local"
+        }
+      },
+      "spec":{
+        "capacity":{
+          "storage" : "8Gi"
+        },
+        "accessModes":[
+          "ReadWriteMany", "ReadWriteOnce",  "ReadOnlyMany"
+        ],
+        "hostPath": {
+          "path" : "/tmp/share"
+        }
+      }
+    },
+    {
+      "kind" : "PersistentVolumeClaim",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "solr-claim"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteMany", "ReadWriteOnce",  "ReadOnlyMany"
+        ],
+        "resources": {
+          "requests": {
+            "storage": "3Gi"
+          }
+        },
+        "selector":{
+          "matchLabels":{
+            "name" : "solr-index-backup",
+            "type" : "local"
+            }
+          }
+        }
+      }
+```
+
+To make solr container mount the hostPath, add the following part under dataverse-solr.spec.spec (for Solr StatefulSet):
+```
+    {
+      "kind": "StatefulSet",
+      "apiVersion": "apps/v1beta1",
+	  "metadata": {
+        "name": "dataverse-solr",
+	  ....
+
+      "spec": {
+        "serviceName" : "dataverse-solr-service",
+		.....
+
+          "spec": {
+            "volumes": [
+              {
+                "name": "solr-index-backup",
+                "persistentVolumeClaim": {
+                  "claimName": "solr-claim"
+                }
+              }
+            ],
+
+			"containers": [
+              ....
+
+                "volumeMounts":[
+                  {
+                    "mountPath" : "/home/share",
+                    "name" : "solr-index-backup"
+                  }  
+
+```
+
+Solr is now ready for backup and recovery. In order to backup:
+
+```
+	oc rsh dataverse-solr-0
+	curl 'http://localhost:8983/solr/collection1/replication?command=backup&location=/home/share'  
+```
+
+In solr entrypoint.sh, it's configured so that if dataverse-solr-0 failed, it will get the latest version of the index in the backup and restore. All backups are store in /tmp/share in the host, or /home/share in solr containers
```

Comment:
maybe use a different path other than /tmp/share in the example.  Perhaps something under /var